src/singleRequestProxyFactory.ts

```
@@ -24,12 +27,15 @@ export function handleERC20SingleRequestProxyCreated(
   event: ERC20SingleRequestProxyCreated,
 ): void {
   let deployment = new SingleRequestProxyDeployment(
-    event.transaction.hash.toHexString(),
+    event.transaction.hash.toHexString() + "-" + event.logIndex.toString(),
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)